### PR TITLE
Build /pro/why-pro page

### PR DESCRIPTION
**Pull request by petesfrench**
## Done

- Build /why-pro page base on [copydoc](https://docs.google.com/document/d/1f1LU-xfBYt48UcJxYcNkW4mGB2u1spdjJrILJBGCI1Y/edit#heading=h.rocxkjlhnbpz) and [design](https://www.figma.com/file/mCV6YajRJtsnEsctbUNgI3/23.10-U.com---Why-Ubuntu-Pro?node-id=158%3A2342&mode=dev)
- Added some custom styling for the `p-divider` class to removed a divider from a specific block, issue create [here](https://github.com/canonical/vanilla-framework/issues/4877)

## QA

- Check out https://ubuntu-com-13182.demos.haus/pro/why-pro compare against [copydoc](https://docs.google.com/document/d/1f1LU-xfBYt48UcJxYcNkW4mGB2u1spdjJrILJBGCI1Y/edit#heading=h.rocxkjlhnbpz) and [design](https://www.figma.com/file/mCV6YajRJtsnEsctbUNgI3/23.10-U.com---Why-Ubuntu-Pro?node-id=158%3A2342&mode=dev)

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-4329

### CODE REVIEW

**Comment by petesfrench** [comment]
@juanruitina I have addressed the text wrapping issue, this has come about from a recent vanilla change and we should keep an eye out for it elsewhere.

The colour difference in the search also comes from vanilla and is applied when the `is-paper` class is used and isn't restricted to this page. We should address this with vanilla if we want it changed. [The vanilla snippet in question](https://github.com/canonical/vanilla-framework/blob/38ffd6939d8d753eb02e02816ff3ac514a976940/scss/_patterns_search-box.scss#L102)

File: templates/ai/what-is-kubeflow.html [hunk 1]
```diff
@@ -8,9 +8,6 @@
 
 <section class="p-strip u-no-padding--top u-no-padding--bottom">
   <div class="p-strip--suru-topped">
-    {% if schedule_banner("2023-03-08", "2023-03-22") %}
-      {% include "shared/_charmed-kubeflow-beta-banner.html" %}
-    {% endif %}
```
Comment (petesfrench) [inline templates/ai/what-is-kubeflow.html:13]: @carkod Can this be removed?

Comment (carkod) [reply templates/ai/what-is-kubeflow.html:13]: I wouldn't remove it, we may need it soon in early 2024 again

**Comment by carkod** [comment]
@lyubomir-popov can you review? Thank you.

**Comment by juanruitina** [comment]
> The colour difference in the search also comes from vanilla and is applied when the `is-paper` class is used and isn't restricted to this page. We should address this with vanilla if we want it changed. [The vanilla snippet in question](https://github.com/canonical/vanilla-framework/blob/38ffd6939d8d753eb02e02816ff3ac514a976940/scss/_patterns_search-box.scss#L102)

Could we apply `is-paper` to the wrapper div only? It should do the trick for a bit, and I think this won't be a problem anymore when we go live with the meganav.

**Comment by lyubomir-popov** [comment]
Looks good, a few small fixes:


- Hero - can we have shallow padding at the top, deep at the bottom - as it is done [here](https://ubuntu.com/ai):
image

Can we have this section like so:
image
(line above headings, not muted, h4 changed to a paragraph, and h4s in the list of 8 items changed to an h5 - with no classes on the h5, just default styling)

u-no-margin--bottom on the small caps heading here:
image

**Comment by petesfrench** [comment]
@lyubomir-popov Can you take another look please

**Comment by lyubomir-popov** [comment]
Looks good, thanks @petesfrench

File: templates/ai/what-is-kubeflow.html (hunk 1 above)
Comment (carkod) [reply templates/ai/what-is-kubeflow.html:13]: And why do you need to change this page? The ticket was for /pro/why-pro?

File: templates/ai/what-is-kubeflow.html [hunk 2]
```diff
@@ -20,7 +17,7 @@ <h1 class="u-sv2">What is Kubeflow?</h1>
           <a class="p-button--positive" href="https://charmed-kubeflow.io/docs/install">
           Charmed Kubeflow
           </a>
-          <a href="https://charmed-kubeflow.io/#get-in-touch">Contact us now&nbsp;&rsaquo;</a>
```
Comment (carkod) [inline templates/ai/what-is-kubeflow.html:23]: I think spaces probably don't have any problems rendering. Just the `&rsaquo;`

**Review by carkod — APPROVED** [review]
LGTM. I asked Pdms (Lech/Gloria) to take a look, since this is a new pro page, if they don't reply by end of the day you can merge this. Thanks

File: templates/ai/what-is-kubeflow.html (hunk 2 above)
Comment (petesfrench) [reply templates/ai/what-is-kubeflow.html:23]: if you don't add a `&nbsp;` it is possible that the  `&rsaquo;` can fall onto the next line on certain screen sizes

**Comment by carkod** [comment]
Good to go! We've got 3 Pdms +1!

**Pull request merged**